stubs/jmespath/jmespath/parser.pyi

```diff
@@ -1,15 +1,18 @@
 from typing import Any
 
+from jmespath.lexer import _LexerTokenizeResults
+from jmespath.visitor import _TreeNode
+
 class Parser:
-    BINDING_POWER: Any
-    tokenizer: Any
+    BINDING_POWER: dict[str, int]
+    tokenizer: _LexerTokenizeResults
```

Comment:
Looks like this can sometimes be `None`: https://github.com/jmespath/jmespath.py/blob/bbe7300c60056f52413603cf3e2bcd0b6afeda3d/jmespath/parser.py#L79

```suggestion
    tokenizer: _LexerTokenizeResults | None
```